python/CMakeLists.txt

```
@@ -2,6 +2,8 @@
 cmake_minimum_required(VERSION 3.10.2)
 project(hyperonpy)
 
+option(GIT "Adds git features to hyperon library; requires OpenSSL" ON)
```

Should `zlib` also be mentioned?
```suggestion
option(GIT "Adds git features to hyperon library; requires OpenSSL and Zlib" ON)
```